protocol-test-util/src/main/java/io/camunda/zeebe/test/broker/protocol/commandapi/PartitionTestClient.java

```diff
@@ -100,7 +100,7 @@ public long deploy(final BpmnModelInstance process) {
 
     TestUtil.waitUntil(
         () ->
-            RecordingExporter.deploymentRecords(DeploymentIntent.FULLY_DISTRIBUTED)
+            RecordingExporter.deploymentRecords(DeploymentIntent.CREATED)
```

Comment:
❌ This method is used in multi-partition scenarios and the change may lead to flaky tests

`FULLY_DISTRIBUTED` and `CREATED` are not equivalent when this method is used in a multi-partition scenario. That is the case in `ExporterManagerPartitionTest.shouldRunExporterForEveryPartition` and in `UniqueKeyFormatTest.shouldStartProcessInstanceAtNoneStartEvent`.

We should either verify that the distribution of the deployment has FINISHED, or better still: verify that Deployment is CREATED on all partitions.